components/profile-controller/controllers/profile_controller.go

```diff
@@ -766,19 +847,24 @@ func setNamespaceLabels(ns *corev1.Namespace, newLabels map[string]string) {
 	}
 }
 
-func (r *ProfileReconciler) readDefaultLabelsFromFile(path string) map[string]string {
-	logger := r.Log.WithName("read-config-file").WithValues("path", path)
-	dat, err := ioutil.ReadFile(path)
-	if err != nil {
-		logger.Error(err, "namespace labels properties file doesn't exist")
-		os.Exit(1)
-	}
-
-	labels := map[string]string{}
-	err = yaml.Unmarshal(dat, &labels)
-	if err != nil {
-		logger.Error(err, "Unable to parse default namespace labels.")
-		os.Exit(1)
+func (r *ProfileReconciler) readDefaultLabelsFromFiles(labelPaths []string) (map[string]string, error) {
+	allLabels := map[string]string{}
+	for _, path := range labelPaths {
+		logger := r.Log.WithName("read-config-file").WithValues("path", path)
+		dat, err := ioutil.ReadFile(path)
+		if err != nil {
+			logger.Error(err, "unable to read default namespace labels file")
```

Comment:
Tweaked these logger messages a little bit so it is clear they are specific to the file in the path key